engine/README.md

```diff
@@ -6,5 +6,7 @@ Flutter engine uses `gclient` to manage dependencies.
 
 If you've already cloned the flutter repository:
 
-1. Copy one of the `engine/scripts/*.gclient` to the root folder as `.gclient`
+1. Copy one of the `engine/scripts/*.gclient` to the root folder as `.gclient`:
+    1. Googlers: copy `rbe.gclient` to enable faster builds with RBE
```

Comment:
I would also link to <https://github.com/flutter/flutter/blob/master/engine/src/flutter/docs/rbe/rbe.md>.

Reply:
googlers could also replace the url with git@github link?

Reply:
Added the link to RBE.